include/drivers/cmn/cmn_clk.h

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2021 Wavious LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef _CMN_CLK_H_
+#define _CMN_CLK_H_
+
+#include <stdbool.h>
+#include "cmn_reg.h"
+
+/**
+ * @brief   Common Clock Control Set PLL0 Clock Divider Enable Register Interface
+ *
+ * @details Sets PLL0 Clock Divider Enable.
```

Comment:
These should be Clock Divider Reset, not Enable

Reply:
good catch.